test/commands/prodly/nut/checkin.nut.js

```diff
@@ -0,0 +1,117 @@
+import { execCmd } from '@salesforce/cli-plugins-testkit';
+import { expect } from 'chai';
+import * as CONST from './utilities/constants.js';
```

Comment:
since "const" in lowercase is reserved word in JS, I will suggest to rename this to something different like CONSTANTS.

Reply:
1. Done 
2.